coop_default_pricetag/models/product_template.py

```diff
@@ -88,8 +88,6 @@ def _compute_pricetag_rackinfos(self):
         help="A product in mercuriale has price"
         " that changes very regularly.",
     )
-    weight_net = fields.Float("Net Weight", default=0)
-
     price_volume = fields.Char(
         compute="_compute_price_volume", string="Price by liter"
     )
```

Comment:
Let's:
- Rename `price_weight_net` to `price_weight`
- Make `price_volume` and `price_weight` Monetary fields instead of Char
- Make them stored

Reply:
Okay @ivantodorovich 

 Noted it.

Reply:
Hi @ivantodorovich 

Changes are pushed, pl. check it out at your convenience.